src/util/Constants.js

```diff
@@ -84,6 +84,7 @@ exports.DefaultOptions = {
     api: 'https://discord.com/api',
     cdn: 'https://cdn.discordapp.com',
     invite: 'https://discord.gg',
+    template: 'https://discord.com/template',
```

Comment:
there needs to be a jsdoc `@property` tag for this too, and the typings need updating for it